content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+So while testing CAT empirically is not a simple matter of yes or no, the theory provides a clear set of tools and lines of thought that are well worth evaluating in a wider, programmatic appraisal.
+
+The second challenge to empirical exploration resides in the fact that quantitative data on out-of-laboratory cultural artefacts is not easy to collect.
+Although theoretical models to guide that exploration are gradually appearing [@claidiere_role_2007; @claidiere_how_2014], developing methods for the quantitative study of attractors is still an open problem.
+The literature on CAT shows at least two important methods that have been used up to now.
+First, the meta-analysis of large numbers of anthropological or historical works, which has uncovered several relevant effects.
+The way portraits are painted over the centuries, for instance, has been show to increasingly favour direct- versus oblique-gaze portraying [@morin_how_2013].
+@miton_universal_2015 also used this technique to propose an explanation for the historical stability of the bloodletting practice, in spite of its medical ineffectiveness.
+Exploiting similar records, @baumard_increased_2015 showed a link between the evolution of religious values and the affluence of societies in which they develop.
+Morin and Miton (EHBEA 2017? \cn), using the same approach, model the evolution of heraldic coats of arms to test which hypotheses account best for the distribution of patterns and colours through time.
+
+A second approach is to reproduce the evolution of content with human or animal participants in the laboratory, where subjects repeatedly transmit or interact around pieces of content initially chosen by the experimenters.
+The resulting data is an approximation of the rapid evolution of artefacts in real life, yet in a controlled situation that can then be statistically analysed with access to all the parameters.
+This technique has been used in studies ranging from the evolution of visual patterns transmitted in a group of apes [@claidiere_cultural_2014], to the role of argumentation in the transmission of solutions to simple but counter-intuitive reasoning problems [@claidiere_argumentation_2017].
+\add{baumard-mutualistic-2013.}
+A number of other disciplines have developed methods that can be relevant to the study of CAT and SCE, and I review these in @sec:intro-review-neighbours.
+
+While authors have acknowledged that there is no core incompatibility between CAT and SCE [@acerbi_if_2015; @sterelny_cultural_2017] and a growing number of social scientists is discussing both approaches positively [see the comments to @mesoudi_towards_2006; and @slingerland_what_2008], the work I present in this thesis is mainly focused on CAT itself, and less so SCE.
```

Comment:
Oui et non, cf. la partie 1.2.1 (ça répond ?)

TODO
- [ ] note that there has been much quarrel about replicative or non-replicative cultural processes